components/search/SearchWidget.vue

```diff
@@ -90,7 +90,7 @@ function activate() {
       </button>
     </div>
     <!-- Results -->
-    <div left-0 top-11 absolute w-full z10 group-focus-within="pointer-events-auto visible" invisible pointer-events-none>
+    <div left-0 top-11 absolute w-full z-10 group-focus-within="pointer-events-auto visible" invisible pointer-events-none>
```

Comment:
This is not related to the fix but all other z-index have the prefix `z-` so use the same format for better code discoverability.